modules/packer/packer_test.go

```diff
@@ -117,3 +118,51 @@ func TestExtractArtifactINoIdPresent(t *testing.T) {
 		t.Error("Expected to get an error when extracting an Artifact ID from text with no Artifact ID in it, but got nil")
 	}
 }
+
+func TestFormatPackerArgs(t *testing.T) {
+	t.Parallel()
+
+	tests := []struct {
+		option   *Options
+		expected string
+	}{
+		{
+			option: &Options{
+				Template: "packer.json",
+			},
+			expected: "build -machine-readable packer.json",
+		},
+		{
+			option: &Options{
+				Template: "packer.json",
+				Vars: map[string]string{
+					"foo": "bar",
+				},
+				Only: "onlythis",
+			},
+			expected: "build -machine-readable -var foo=bar -only=onlythis packer.json",
+		},
+		{
+			option: &Options{
+				Template: "packer.json",
+				Vars: map[string]string{
+					"foo": "bar",
+				},
+				VarFiles: []string{
+					"foofile.json",
+				},
+			},
+			expected: "build -machine-readable -var foo=bar -var-file foofile.json packer.json",
+		},
+	}
+
+	for _, test := range tests {
+		args := formatPackerArgs(test.option)
+		if strings.Join(args, " ") != test.expected {
```

Comment:
You can simplify this to `assert.Equal(t, strings.Join(args, " "), test.expected)`. When it is not equal, assert will show the expected and actual as well.

Reply:
The import is `github.com/stretchr/testify/assert`

Reply:
Thanks a lot for that suggestion! I updated it to use `assert.Equal`.